storybook/mocks/wallet.ts

```diff
@@ -536,6 +540,54 @@ const fetchCurrentPrice = {
   },
 }
 
+const fetchTokenInfo = {
+  nft: async (...args) => {
```

Comment:
wrap within the `success`, also if there is a story for `loading`. I missed the success with `base64` on `logo` property
`{ success {nft, ft, ftNoImage }, loading {...}}...`